packages/internal-helpers/src/path.ts

```diff
@@ -80,6 +80,11 @@ export function removeFileExtension(path: string) {
 	return idx === -1 ? path : path.slice(0, idx);
 }
 
+export function getFileExtension(path: string) {
+	let idx = path.lastIndexOf('.');
+	return idx === -1 ? '' : path.slice(idx+1);
+}
```

Comment:
Node provides a functionality to retrieve the file extension from a string: https://nodejs.org/dist/latest-v20.x/docs/api/path.html#pathextnamepath

Reply:
I have replaced it with `extname`